.circleci/config.yml

```diff
@@ -1,23 +1,5 @@
 version: 2
 jobs:
-  test-node-8:
```

Comment:
I dont think we should do this.
atem-connection needs to work with node8 for companion, and so needs threadedClass to continue to work.
We have added an `yarn install --ignore-engines` to allow us to use some newer dependencies to keep the maintenance cost down